src/pages/verify/integrations/react.mdx

```diff
@@ -8,6 +8,10 @@ subtitle: Accept MitID, NemID, Swedish BankID, Norwegian BankID and more logins
 import Layout from '../../../layouts/mdx';
 export default Layout;
 
+This guide shows you how to use Criipto Verify in a React web application. 
+
+Read the [full documentation](https://github.com/criipto/criipto-verify-react) or [download a sample application from GitHub](https://github.com/criipto/criipto-verify-react/tree/master/example).
```

Comment:
Might need to save some scrolling space when it's embedded on the dashboard, so perhaps have both lines in the same paragraph?

Reply:
Sure, thanks.